test/test_manifest.json

```diff
@@ -2080,5 +2080,11 @@
        "md5": "d5b37f8bf1b3aafa1b4fcf19ebdc7c74",
        "rounds": 1,
        "type": "eq"
+    },
+    {  "id": "issue5909",
+       "file": "pdfs/issue5909.pdf",
+       "md5": "65c169b6f540b27ac0ff2738a80d1e14",
+       "rounds": 1,
+       "type": "eq"
```

Comment:
Ok, yeah I can seeing as it's just a blank PDF - it's not going to render anything anyway. I'll make that change and push up a new commit.